examples/index.html

```diff
@@ -659,9 +659,5 @@ <h3>Output</h3>
     // also available : 'setFilters'
   });
 </script>
-
-<script>
-  document.write('<script src="//' + location.host.split(':')[0] + ':35729/livereload.js" async defer><' + '/script>');
-</script>
```

Comment:
This snippet appears to come from the [live-server docs](https://www.npmjs.com/package/livereload#method-1-use-browser-extension), but the project appears to be using [alive-server](https://www.npmjs.com/package/alive-server) which works differently

> You don't need to install any browser plugins or manually add code snippets to your pages for the reload functionality to work